model/src/main/java/org/openremote/model/teltonika/TeltonikaDataPayload.java

```diff
@@ -0,0 +1,16 @@
+package org.openremote.model.teltonika;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonPropertyOrder({
+        "payload"
+})
+public class TeltonikaDataPayload {
+
+    @JsonProperty("state")
+    public State state;
```

Comment:
Fields are by default used as json properties so this isn't needed unless you want to change the key name